From ef2ba9936e82a41bf27538fab0dc7ab0fff4c4bf Mon Sep 17 00:00:00 2001
From: daveyannihilation <daveyannihilation@hotmail.com>
Date: Sun, 1 Jan 2017 01:47:53 -0700
Subject: [PATCH 32/35] SystemUI: Expose switch bar title

This is needed for the power notifications switchbar in SystemUI Tuner, amongst other things.

Change-Id: I86f04840c2be46519509556b8d0061cefe26f631
---
 packages/SystemUI/res/layout/switch_bar.xml     |  3 +--
 packages/SystemUI/res/values/projekt_styles.xml | 24 ++++++++++++++++++++++++
 2 files changed, 25 insertions(+), 2 deletions(-)
 create mode 100644 packages/SystemUI/res/values/projekt_styles.xml

diff --git a/packages/SystemUI/res/layout/switch_bar.xml b/packages/SystemUI/res/layout/switch_bar.xml
index 41cdb78..344c5aa 100644
--- a/packages/SystemUI/res/layout/switch_bar.xml
+++ b/packages/SystemUI/res/layout/switch_bar.xml
@@ -33,8 +33,7 @@
         android:paddingStart="48dp"
         android:maxLines="2"
         android:ellipsize="end"
-        android:textAppearance="@android:style/TextAppearance.Material.Title"
-        android:textColor="?android:attr/textColorPrimaryInverse"
+        android:textAppearance="@style/TextAppearance.SwitchBar"
         android:textAlignment="viewStart"
         android:text="@string/switch_bar_on" />
 
diff --git a/packages/SystemUI/res/values/projekt_styles.xml b/packages/SystemUI/res/values/projekt_styles.xml
new file mode 100644
index 0000000..f49834b
--- /dev/null
+++ b/packages/SystemUI/res/values/projekt_styles.xml
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (c) 2016 Projekt Substratum
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<resources>
+    <!-- Switch bar text appearance -->
+    <style name="TextAppearance.SwitchBar" parent="@android:style/TextAppearance.Material.Title">
+        <item name="android:textColor">?android:attr/textColorPrimaryInverse</item>
+    </style>
+</resources>
+
-- 
2.7.4

